src/Plugin/Collection/PdkOrderCollection.php

```diff
@@ -45,6 +45,24 @@ public function getAllShipments(): ShipmentCollection
         }, new ShipmentCollection());
     }
 
+    /**
+     * @return \MyParcelNL\Pdk\Shipment\Model\Shipment[]
+     */
+    public function getLastShipments(): array
```

Comment:
ik snap niet waar dit voor nodig is, kun je de use case hiervoor uitleggen?

Reply:
De use case is het huidige 'print labels'. commando, die van alle orders de meeste recente label uitprint (indien aanwezig). Zonder deze method zou je dan van de gekozen orders alleen alle labels kunnen uitprinten.